tests/cases/fourslash/findAllRefsForVariableInExtendsClause01.ts

```diff
@@ -3,4 +3,4 @@
 ////var [|{| "isWriteAccess": true, "isDefinition": true |}Base|] = class { };
 ////class C extends [|Base|] { }
 
-verify.singleReferenceGroup("var Base: typeof Base");
+verify.singleReferenceGroup("var Base: {\n    new (): {};\n    prototype: {};\n}");

```

Comment:
If there is a way to keep this as typeof Base it would be better I think. The completionlist/quickinfoes would get unreadable with this change otherwise